config.json

```diff
@@ -15,14 +15,29 @@
     "indent_size": 2
   },
   "files": {
-    "solution": [],
-    "test": [],
-    "example": [],
+    "solution": [
+      "%{kebab_slug}.ys"
+    ],
+    "test": [
+      "%{kebab_slug}-test.ys"
+    ],
+    "example": [
+      ".meta/%{kebab_slug}.ys"
+    ],
     "exemplar": []
   },
   "exercises": {
     "concept": [],
-    "practice": []
+    "practice": [
+      {
+        "slug": "hello-world",
+        "name": "Hello World",
+        "uuid": "f4c93e64-d954-4d2d-b4c9-d27d87a3390f",
```

Comment:
How was this calculated?
Can you point to the documentation for this?

Reply:
It simply needs to be a newly generated uuid, never copy a uuid from another track or another exercise.

If you have run `bin/fetch-configlet` then you can run `bin/configlet uuid` to generate a new uuid.

Another option when beginning to add a new exercise is to run `bin/configlet create --practice-exercise ...`  and it will update config.json for you, as well as creating a couple of other files for you, e.g. tests.yaml 

Another option is that you implement in https://github.com/exercism/yamlscript/tree/main/bin a track-specific script like   [create-exercise.sh](https://github.com/exercism/idris/blob/main/bin/create-exercise.sh)   that would call `bin/configlet create` and also create any track-specific files that you would like each exercise to have.